.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,6 @@
+blank_issues_enabled: true
+
+contact_links:
+  - name: "\u2753 Discord Community"
+    url: "https://discord.gg/nestjs"
+    about: "Feel free to use NestJS's Discord server for a more in-depth support."
```

Comment:
I usually tell people to create an issue here for an actual in-depth support :smile: 

This should probably say something "if you're not sure you have an actual issue and just want to chat about it, visit the NestJS discord" or something. I don't think Discord is a good place to provide in-depth support, given its ephemeral nature.